src/idas/idas_impl.h

```diff
@@ -977,6 +978,11 @@ int IDASensResDQ(int Ns, sunrealtype t, N_Vector yy, N_Vector yp,
                  N_Vector* resvalS, void* user_dataS, N_Vector ytemp,
                  N_Vector yptemp, N_Vector restemp);
 
+/* Utility function to tell IDA to free the user data.
+   This is used by the Python interfaces. */
+
+int IDASetOwnUserData(void* ida_mem, sunbooleantype own_user_data);
```

Comment:
IDA is inconsistent about this but to follow the general convention prefix with `ida`
```suggestion
int idaSetOwnUserData(void* ida_mem, sunbooleantype own_user_data);
```